Homeworks/Homework_1/task_2.py

```diff
@@ -0,0 +1,22 @@
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+# Дано a, b, c - стороны предполагаемого треугольника.
+# Требуется сравнить длину каждого отрезка-стороны с суммой двух других.
+# Если хотя бы в одном случае отрезок окажется больше суммы двух других,
+# то треугольника с такими сторонами не существует.
+# Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+def triangle(a, b, c):
+    if a + b > c and b + c > a and a + c > b:
+        if a == b == c:
+            print("Треугольник равносторонний")
+        elif a != b and b != c and c != a:
+            print('Треугольник разносторонний')
+        elif a == b or b == c or c == a:
```

Comment:
А это уже не требуется, так как ничего больше не осталось
```suggestion
        else:
```